website/content/docs/connect/config-entries/service-splitter.mdx

```diff
@@ -1,54 +1,567 @@
+--- 
+Page_title: Service Splitter | Configuration Reference
+Description: Service splitter config entries are L7 traffic management tools that enable Consul to redirect parts of incoming traffic requests for a specific service. 
 ---
-layout: docs
-page_title: Service Splitter - Configuration Entry Reference
-description: >-
-  The service splitter configuration entry kind defines how to divide service mesh traffic between service instances. Use the reference guide to learn about `""service-splitter""` config entry parameters and how it can be used for traffic management behaviors like canary rollouts, blue green deployment, and load balancing across environments.
----
 
-# Service Splitter Configuration Entry 
+# Service Splitter Configuration Reference
+
+This topic describes how to configure service splitter configuration entries. Configure and apply service splitters to redirect a percentage of incoming traffic requests for a specific service.
```

Comment:
1. I like "topic" as a general way to talk about a unit of information, but I don't have a problem with "reference page". 
2. My thinking is that it might be clearer to refer to the "service splitter configuration entry" when talking about the entity in general and use `service-splitter` and `ServiceSplitter` when referring to the VM and YAML configurations respectively.